gradle/verification-metadata.xml

```diff
@@ -27,6 +27,7 @@
          <trusted-key id="314fe82e5a4c5377bca2edec5208812e1e4a6db0" group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.12.3"/>
          <trusted-key id="34441e504a937f43eb0daef96a65176a0fb1cd0b" group="org.codehaus.groovy" name="^groovy($|.*)" version="3.0.13" regex="true"/>
          <trusted-key id="475f3b8e59e6e63aa78067482c7b12f2a511e325" group="org.slf4j"/>
+         <trusted-key id="e2acb037933cdeaab7bf77d49a2c7a98e457c53d" group="org.springframework" name="spring-framework-bom" version="5.3.24"/>
```

Comment:
Why is this dependency added?

Reply:
Might be a transient dependency of Shadow plugin.